### ISSUE TITLE
Stats: Improve contrast for Last Update on Stats page

### ISSUE THREAD

**Issue by sixhours** [Automattic/wp-calypso#33757]
Found while working on color schemes for Calypso. The "Last Update" line of text doesn't meet WCAG AA standards for contrast and could be improved. Switching to `var( --color-text-subtle )` from `var( --color-neutral-light )` would fix it.

<img width="369" alt="Screen Shot 2019-06-07 at 1 40 10 PM" src="https://user-images.githubusercontent.com/2124984/59123487-4eca7180-892b-11e9-85d6-b2ddf07cda8a.png">

**Comment by sixhours** [comment]
This was fixed in #33917
